sn_node_manager/src/bin/cli/main.rs

```diff
@@ -547,6 +547,9 @@ pub enum LocalSubCmd {
         /// Set to skip the network validation process
         #[clap(long)]
         skip_validation: bool,
+        /// Specify the owner(readable discord user name).
+        #[clap(long)]
+        owner: Option<String>,
```

Comment:
This is just for a local network and I assue we might need to pass in the `onwer` inside the `Add` arg to enable this for service nodes.

Reply:
as discussed, this can be addressed in follow up PR